### ISSUE TITLE
yolov3 training

### ISSUE THREAD

**Issue by anguoyang** [AlexeyAB/Yolo_mark#37]
@AlexeyAB it is exciting to see that yolo v3 is coming:)
could this tool be used to train v3? I noticed that the configuration is different, could we keep the filters num as V2? thank you

**Comment by AlexeyAB** [comment]
@anguoyang 
Yes, this tool can be used for both Yolo v2 and Yolo v3 in the same way.
I changed manual How to train for Yolo v3: https://github.com/AlexeyAB/darknet#how-to-train-to-detect-your-custom-objects

**Comment by anguoyang** [comment]
Hi@AlexeyAB , thank you so much!

**Comment by anguoyang** [comment]
@AlexeyAB .I followed the above steps, but with these errors:
Unused field: 'mask = 6,7,8'
Unused field: 'anchors = 10,13,16,30,33,23,30,61,62,45,59,119,116,90,156,198,373,326'
Unused field: 'classes = 10'
Unused field: 'num = 9'
Unused field: 'jitter = .3'
Unused field: 'ignore_thresh = .5'
Unused field: 'truth_thresh = 1'
Unused field: 'random = 1'
   83 route  79
   84 conv    256  1 x 1 / 1    13 x  13 x 512   ->    13 x  13 x 256
   85 Type not recognized: [upsample]
Unused field: 'stride = 2'
   86 route  85 61
   87 Layer before convolutional layer must output image.: No error

**Comment by AlexeyAB** [comment]
@anguoyang Update your code from this repo.

**Comment by anguoyang** [comment]
@AlexeyAB thank you, now I updated and trained the yolov3 successfully, however, when I run the darknet_yolo_v3_video.cmd, there is only bounding box, no confidence on it, could you please tell me which param need to be used for displaying the confidence?
